manifests/params.pp

```diff
@@ -182,7 +182,7 @@
   $ldap_clientcert                          = undef
   $ldap_clientkey                           = undef
   $ldap_reqcert                             = undef
-  $server_api_pass                          = 'zabbix'
+  $server_api_pass                          = Sensitive('zabbix')
```

Comment:
I’m not sure if we must set default values as sensitive.

I’ve changed some tests because of this: if the default value is `Sensitive`, the output may be masked during tests, and the validation cannot be made.  These lines are marked with this comment below:

```
# cleartext password must be explicitly declared in this test, otherwise the parser will secure content of the file
```

For example, see `spec/classes/proxy_spec.rb:115`

Reply:
IMO that's okay